contracts/src/interfaces/IEigenDABlobVerifier.sol

```diff
@@ -29,5 +29,12 @@ interface IEigenDABlobVerifier is IEigenDAThresholdRegistry {
         bytes calldata additionalQuorumNumbersRequired
     ) external view;
 
-    function verifyBlobV2() external view;
-}
\ No newline at end of file
+    function verifyBlobV2(
+        EigenDABlobVerificationUtils.SignedCertificate calldata signedCertificate
+    ) external view;
+
+    function verifyBlobV2(
+        EigenDABlobVerificationUtils.SignedCertificate calldata signedCertificate,
+        bytes calldata additionalQuorumNumbersRequired
```

Comment:
blob header (contained in blob cert) contains list of all quorums. Can we infer quorum numbers from the blob header? Then we can just have one interface above